src/components/Home/StepperSection.tsx

```diff
@@ -0,0 +1,356 @@
+              }}
+            >
+              You can start by requesting your personal data from Google and
+              Facebook.{' '}
+            </Typography>
+            <Typography
+              sx={{
+                lineHeight: '2rem',
+                fontWeight: 400,
+                fontSize: '1.2rem',
+              }}
+            >
+              It can take up to 48h to receive your data files : Facebook will
+              send you a notification and Google an e-mail when your files are
+              ready to be downloaded.{' '}
+            </Typography>
+            <Typography
+              sx={{
+                lineHeight: '2rem',
+                fontWeight: 400,
+                fontSize: '1.2rem',
+              }}
+            >
+              At the end of this step, you should have received multiple files
+              in Zip format. Download them locally on your device.
+            </Typography>
+          </Box>
+          <Box width={'100%'}>
+            <video width="100%" height="auto" controls>
+              <source
+                src={require('../../assets/video/video-test.mp4')}
+                type="video/mp4"
+              />
+            </video>
+          </Box>
+        </VideoSection>
+      );
+    case 1:
+      return (
+        <DropZoneSection>
+          <Box
+            display="flex"
+            flexDirection="column"
+            justifyContent="center"
+            gap="0.813rem"
+          >
+            <Typography
+              sx={{
+                lineHeight: '2rem',
+                fontWeight: 400,
+                fontSize: '1.2rem',
+              }}
+            >
+              Once you’ve received your data, you can initialize your Avastar
+              digital identity card. Upload your data files right here and
+              finally access what you always wanted to know about your personal
+              data !
+            </Typography>
+            <Typography
+              sx={{
+                lineHeight: '2rem',
+                fontWeight: 400,
+                fontSize: '1.2rem',
+              }}
+            >
+              The data you upload will not reach any server. The files you
+              upload never leave your computer, all your data is stored locally
+              on your browser. It will also be deleted every time your session
+              is refreshed
+            </Typography>
+          </Box>
+          <Box display="flex" alignItems="flex-start" gap="3.875rem">
+            <Box>
+              <IconContainer>
+                <Icon src={GoogleIcon} alt={GoogleIcon} />
+              </IconContainer>
+              <DropZone />
+            </Box>
+            <Box>
+              <IconContainer>
+                <Icon src={FacebookIcon} alt={FacebookIcon} />
+              </IconContainer>
+              <DropZone />
+            </Box>
+          </Box>
+        </DropZoneSection>
+      );
+    case 2:
+      return (
```

Comment:
Dans cette section, il faut que je te transfère les infos suivantes pour qu'elle soit finalisée : 
* Les 3 paragraphes de texte ; 
* Le GIF de l'overview page.

Je te forward ça dès que j'ai terminé.

Reply:
J'ai drafté ce texte pour la section 3 du stepper : 

"Visualise what kind of information have been collected by apps and websites about you since you got started on the internet.

For each visualisation, we've included explanations to help you decide with whom and what data you should share on the internet.

You will have the keys to remove access to entities from which you want to protect your data and identify sensitive information that may harm your online reputation."

Reply:
Je trouve ce texte suffisamment explicite, je l'ai d'ores et déjà intégré à la section 3 du stepper du coup :+1: